extending-valtimo/tabs/configure-tab-object-type.md

```diff
@@ -0,0 +1,35 @@
+# Setting extra tabs for Zaak object type
+
+On the details page of a case instance, several tabs are shown, including the Zaakobject tab.
```

Comment:
I would remove this, the Zaakobjecten tab is not shown by default, but can be configured.